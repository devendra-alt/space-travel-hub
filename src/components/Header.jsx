import React from 'react';

function Header() {
  return <nav className="main-header-nav"></nav>;
}

export default Header;
