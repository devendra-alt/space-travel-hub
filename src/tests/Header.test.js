import '@testing-library/jest-dom';
import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Header from '../components/Header';

test('Header component renders correctly', () => {
  const { asFragment } = render(
    <BrowserRouter>
      <Header />
    </BrowserRouter>,
  );
  expect(asFragment()).toMatchSnapshot();
});
