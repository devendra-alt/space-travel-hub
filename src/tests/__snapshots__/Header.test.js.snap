// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders correctly 1`] = `
<DocumentFragment>
  <header
    class="main-header"
  >
    <div
      class="site-title-section"
    >
      <img
        alt="space travel hub logo"
        height="60px"
        src="test-file-stub"
        width="60px"
      />
      <h1
        class="site-title"
      >
        Space Traveler's Hub
      </h1>
    </div>
    <nav
      class="main-header-nav"
    >
      <a
        aria-current="page"
        class="nav-link active"
        href="/"
      >
        Rockets
      </a>
      <a
        class="nav-link"
        href="/missions"
      >
        Missions
      </a>
      <a
        class="nav-link"
        href="/my-profile"
      >
        Profile
      </a>
    </nav>
  </header>
</DocumentFragment>
`;
